physical_properties/pKa/analysis/analysis_of_typeI_predictions/README.md

```diff
@@ -0,0 +1,78 @@
+# Analysis of the type I microscopic pKa predictions
+
+General analysis of typeI microscopic pKa predictions include calculated vs predicted pKa correlation plots and 5 performance statistics (RMSE, MAE, ME, R^2 and linear regression slope(m) ) for all the submissions. 
+95%-percentile bootstrap confidence intervals of all the statistics were reported. 
+
+In this analysis approach we ignored stated microstate IDs and paired predicted and experimental pKas based on pKa values only using ith closest and hungarian matching. 
```

Comment:
"ith closest and Hungarian matching"? What does that mean?

Would be nice if there were a clearer overall summary of how this is different from what came before.

Reply:
I updated the README to start with a description of the difference:

"In the absence of direct experimental measurement of microscopic pKas, the analysis of microscopic pKa predictions was performed with the assumption that experimentally determined pKas of molecules with only 1 pKa or well separated multiple pKas (more than 3 pKa units apart) are equal to microscopic pKas. Type I predictions of molecules SM14 and SM18 were excluded from this analysis, since their experimental pKa values don't satisfy these criteria.

In this microscopic pKa value based analysis approach, reported microstate ID pairs were not considered. Instead predicted and experimental pKas were paired based on similarity of pKa values using closest and hungarian matching algorithms."